Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM alpine:latest
+RUN apk --no-cache add curl jq && \
+    SDK_VERSION=$(curl -sL https://api.github.com/repos/snyk/snyk-iac-rules/releases/latest | jq -r ".tag_name" | cut -c2-) && \
+    mkdir ./app && \
+    cd ./app && \
+    wget https://github.com/snyk/snyk-iac-rules/releases/latest/download/snyk-iac-rules_${SDK_VERSION}_Linux_x86_64.tar.gz && \
+    tar -xf snyk-iac-rules_${SDK_VERSION}_Linux_x86_64.tar.gz && \
+    rm snyk-iac-rules_${SDK_VERSION}_Linux_x86_64.tar.gz && \
+    chmod +x ./snyk-iac-rules

```

Comment:
I just want to ensure that this doesn't get lost, instead of doing this fetching of the rules binary from the release, can we look into just copying the generated binary from the local filesystem? I think that would greatly simplify this code.

https://github.com/snyk/snyk-iac-rules/blob/develop/.github/workflows/release.yml#L57-L64

If we want to ensure this can be run locally too then I'd suggest we move this step into a Makefile/bash script.

Reply:
Thanks for this suggestion, I agree with you that your approach will be potentially more easier to implement.
I tried giving it a shot and locally it worked for me without any problem but when I tested it as part of a job inside GH Actions it failed when he tried to COPY the binary in the docker build stage.
I debugged it and it looks like it can't locate the file, I tried doing several additional tests (my last test included changing the target file path to the one I saw when debugging it) but none of them seem to work.
Any chance you might know what can cause this? I'm probably missing something small that I didn't notice...

Reply:
@YairZ101 looking at the output from the Github action, the docker file step creates a temporary directory, copys the files in the dist folder into it, then runs the docker command from that directory. I've just pushed a change to see if it fixes it.

Reply:
Thx for the help, I missed this small line :|
![image](https://user-images.githubusercontent.com/71096571/146245725-8b04433e-c2d3-438d-b847-1ccff01cca55.png)